pkg/kubelet/util/util_unix_test.go

```diff
@@ -69,3 +72,69 @@ func TestParseEndpoint(t *testing.T) {
 	}
 
 }
+
+func TestIsUnixDomainSocket(t *testing.T) {
+	tests := []struct {
+		label          string
+		listenOnSocket bool
+		expectSocket   bool
+		expectError    bool
+		invalidFile    bool
+	}{
+		{
+			label:          "Domain Socket file",
+			listenOnSocket: true,
+			expectSocket:   true,
+			expectError:    false,
+		},
+		{
+			label:       "Non Existent file",
+			invalidFile: true,
+			expectError: true,
+		},
+		{
+			label:          "Regular file",
+			listenOnSocket: false,
+			expectSocket:   false,
+			expectError:    false,
+		},
+	}
+	for _, test := range tests {
+		f, err := ioutil.TempFile("", "test-domain-socket")
+		if err != nil {
```

Comment:
nit: import `github.com/stretchr/testify/require` and use `require.NoError`

Applies to all the unit tests changes in this PR :-)

Reply:
Done.